arch/arm64/src/imx9/imx9_boot.c

```diff
@@ -111,7 +136,9 @@ void arm64_chip_boot(void)
 #endif
 
 #if defined(CONFIG_SMP) || defined(CONFIG_ARCH_HAVE_PSCI)
+#ifndef CONFIG_IMX9_BOOTLOADER
```

Comment:
This flag seems a bit misplaced, this is generic arm64 code but the flag is imx9 specific, I think some other solution is needed to flag this out ?

Reply:
Yes, did it in hurry as noticed that call will crash (didn't go find out why)

Reply:
It goes to call smc etc, but being in EL3 already doesn't make much sense for such.. will go through

Reply:
Removed it, made imx9 select PSCI only if !ARCH_BOOT_EL3